packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -87,20 +93,54 @@ const QueueRoomGroup = styled.div`
   align-items: center;
   margin-bottom: 24px;
 `;
+
+const DisableQueueButton = styled(QueueInfoColumnButton)`
+  color: white;
+  background: #da3236;
+  bottom: 0;
+  position: absolute;
+  &:hover,
+  &:focus {
+    color: white;
+    background: #f76c6c;
+  }
+`;
+
 interface QueueInfoColumnProps {
   queueId: number;
+  isTA: boolean;
   buttons: ReactNode;
 }
 
 export function QueueInfoColumn({
   queueId,
+  isTA,
   buttons,
 }: QueueInfoColumnProps): ReactElement {
-  const { queue } = useQueue(queueId);
+  const { queue, mutateQueue } = useQueue(queueId);
+
+  const disableQueue = async () => {
+    await API.queues.disable(queueId);
+    await mutateQueue(); // cope
+    message.success("Successfully disabled queue: " + queue.room);
+    await Router.push("/");
+  };
+
+  const confirmDisable = () => {
+    confirm({
+      title: `Please Confirm!`,
+      icon: <ExclamationCircleOutlined />,
+      content: `Please confirm that you want to disable the queue: ${queue.room}`,
```

Comment:
Let's add another sentence to this modal to say something like, "This queue will no longer appear in the app, and any students currently in the queue will be removed." so users know what happens upon disabling.